CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## 0.13.0 / 2022-3-14
+- [FEATURE] Add datadog_output type. #149
+- [FEATURE] Add main workflow actions. #152
+- [FEATURE] Add support for rewrite tag. #155
+- [FEATURE] Add aws filter plugin. #173
+- [FEATURE] Add multiline filter plugin. #176
+- [FEATURE] Firehose plugin support. #178
+- [FEATURE] Add volume crd. #186
+- [ENHANCEMENT] Add priority class to FluentBit type. #146
+- [ENHANCEMENT] Upgrade layout from Kubebuilder v2 to Kubebuilder v3.1. #147
+- [BUGFIX] Fix some bugs.
+
```

Comment:
This part can copy from v1.0.0 release docs. Ref: https://github.com/fluent/fluent-operator/blob/release-1.0/CHANGELOG.md#features

Reply:
Thanks, I've amended it a bit. But this is a RELEASE of fluentbit-operator v0.13.0, not of fluentd-operator. So it doesn't add the part about fluentd-operator.